pkl-core/src/main/java/org/pkl/core/stdlib/base/MappingNodes.java

```diff
@@ -73,6 +75,28 @@ protected VmSet eval(VmMapping self) {
     }
   }
 
+  public abstract static class values extends ExternalPropertyNode {
+    @Specialization
+    protected VmList eval(VmMapping self) {
+      var builder = VmList.newBuilder();
+      for (var key : self.getAllKeys()) {
+        builder.add(VmUtils.readMember(self, key));
+      }
+      return builder.build();
```

Comment:
I'm thinking if there's no way to keep this (and `entries`) lazy. Making the `VmList` just a delegate to the map values. Problem I see is the backing `EconomicMap` has just an iterator to the values, so you can't access them by index...

Reply:
A natural way to keep `values` and `entries` lazy would be to return a Listing that delegates to the Mapping.
These more complicated properties/methods could be tackled in a separate PR.
Perhaps `toMap()` should remain the only way to directly transition to eager collections.
(`values.toList()` and `entries.toList()` could also be used.)